learntools/embeddings/nbs/1-embeddings.ipynb

```diff
@@ -574,6 +593,7 @@
     "\n",
     "How should we handle these sparse categorical inputs? Let's start by mentioning a couple reasonable-sounding approaches that won't work here.\n",
     "\n",
+    "#### DB: Can we move the Bad ideas to ThoughtQuestions in the exercise.  Asking the user to identify why X is this a bad idea?\n",
```

Comment:
I'm inclined to keep this part to make it clear why we need embeddings. Again, I think the lesson is more likely to stick if we follow this journey of trying to solve some problem, getting stuck, and then learning about X as a way to overcome the blocker.

Reply:
Maybe move ID's approach to ThoughtQuestion... TBD depending on exercise structure